clang/test/Driver/sycl-offload-intelfpga-link.cpp

```diff
@@ -6,9 +6,9 @@
 // RUN:  | FileCheck -check-prefixes=CHK-FPGA-LINK,CHK-FPGA-EARLY %s
```

Comment:
Does RUN at L5 require `check-prefix` CHK-FPGA-LINK?
Same for RUN at L3.

Reply:
Yes, `CHK-FPGA-LINK` usage is generic in nature.  The `-fsycl-link=early` and `-fsycl-link=image` behaviors are now more closely matched and the `CHK-FPGA-LINK` checks cover the lines that are common.

Reply:
nit: I think `-O2` is probably not needed at L5